sirepo/pkcli/roles.py

```diff
@@ -70,10 +70,9 @@ def _parse_args(uid_or_email, roles):
 
     # POSIT: Uid's are from the base62 charset so an '@' implies an email.
     if '@' in uid_or_email:
-        m = sirepo.auth.get_module('email')
-        assert m, \
-            f'uid_or_email={uid_or_email} contains "@" but email not configured'
-        u = m.unchecked_user_by_user_name(uid_or_email)
+        u = sirepo.auth.get_module(
```

Comment:
why remove the assert?

Reply:
A [KeyError will be raised](https://github.com/radiasoft/sirepo/blob/d2624a6728449a9217c991016f81eac872e94b2d/sirepo/auth/__init__.py#L151) if the module isn't found